Misc/NEWS.d/next/Library/2023-09-02-18-48-44.gh-issue-108826.KG7abS.rst

```diff
@@ -0,0 +1 @@
+:mod:`dis` module command line interface is now mentioned in documentation.
```

Comment:
I am not sure whether the Library section or the Documentation section are more suitable for this entry.

Reply:
Different docs mention CLI in different sections of file. I was looking at `trace` docs when making these changes.

Yes, "Documentation" section seems more suitable, even if I made changes in library code. I'll submit a new entry.